cli/azd/cmd/monitor.go

```diff
@@ -138,9 +140,24 @@ func (m *monitorAction) Run(ctx context.Context) error {
 	}
 
 	openWithDefaultBrowser := func(url string) {
-		fmt.Fprintf(m.console.Handles().Stdout, "Opening %s in the default browser...\n", url)
+		// In Codespaces and devcontainers a $BROWSER environment variable is
+		// present whose value is an executable that launches the browser when
+		// called with the form:
+		// $BROWSER <url>
+
+		const BrowserEnvVarName = "BROWSER"
 
-		if err := open.Open(url); err != nil {
+		if envBrowser := os.Getenv(BrowserEnvVarName); len(envBrowser) > 0 {
+			fmt.Fprintf(m.console.Handles().Stdout, "Opening %s in the browser configured by $BROWSER...\n", url)
```

Comment:
I wonder if we should just reuse the message `Opening %s in the default browser...\n` instead of using a message that has `$BROWSER` in the text.  I suspect that most folks in Codespaces/DevContainers won't know about this variable and it feels pretty "techy".  If we were concerned about the fact that word "default" may be confusing in the case where `$BROWSER` is set, I still think something like `Opening %s in your browser...\n` would be better?

Reply:
The text you're seeing is written purely for me to differentiate what was happening during the command. I should probably use debug logging for that going forward. Indeed the default text is going to be easier for the end user. Thanks!